upgrade/sql/8.0.0.sql

```diff
@@ -203,14 +203,14 @@ ALTER TABLE `PREFIX_webservice_permission` MODIFY COLUMN `method` ENUM(
     'GET', 'POST', 'PUT', 'PATCH', 'DELETE', 'HEAD'
 ) NOT NULL;
 
-ALTER TABLE `PREFIX_product` ADD `unit_price` decimal(20, 6) NOT NULL DEFAULT '0.000000' AFTER `unity`;
-ALTER TABLE `PREFIX_product_shop` ADD `unit_price` decimal(20, 6) NOT NULL DEFAULT '0.000000' AFTER `unity`;
+/* PHP:add_column('product', 'unit_price', "decimal(20, 6) NOT NULL DEFAULT '0.000000' AFTER `unity`"); */;
+/* PHP:add_column('product_shop', 'unit_price', "decimal(20, 6) NOT NULL DEFAULT '0.000000' AFTER `unity`"); */;
 
 UPDATE `PREFIX_product` SET `unit_price` = IF (`unit_price_ratio` != 0, `price` / `unit_price_ratio`, 0);
 UPDATE `PREFIX_product_shop` SET `unit_price` = IF (`unit_price_ratio` != 0, `price` / `unit_price_ratio`, 0);
 
+/* PHP:add_column('feature_flag', 'stability', "VARCHAR(64) DEFAULT 'beta' NOT NULL"); */;
```

Comment:
This doesn't seem to work, from CI :
![a536f11bad6852d3615699fa5dd7eccdfbabc4f2](https://github.com/user-attachments/assets/34006127-c0c1-4ff5-91c4-7916e4ac1b93)

Reply:
I test locally to see the state of the DB

Reply:
I tried without the double quotes, and it works... the handling of calls in CoreUpgrader::runPhpQuery seems to be a problem? In any case, collums are not created

Reply:
@M0rgan01 Hi, could you please check it and see why the stability query doesn't work? :( I used escaped quotes everywhere, but it did not fix it.

Reply:
I'm trying to find a solution

Reply:
@Hlavtox Good news, https://github.com/PrestaShop/autoupgrade/pull/874 solves our problem